tox.ini

```diff
@@ -47,6 +47,10 @@ filterwarnings =
     # Usage of deprecated method of urllib3 within the elasticsearch library
     ignore:^HTTPResponse.getheaders\(\) is deprecated and will be removed in urllib3 v2.1.0. Instead access HTTPResponse.headers directly.$:DeprecationWarning:elasticsearch
 
+    # Ignore a warning triggered by pytest-cov that's out of our control.
+    # See https://github.com/pytest-dev/pytest-xdist/issues/840.
+    ignore:^The --rsyncdir command line argument and rsyncdirs config variable are deprecated\.:DeprecationWarning
+
```

Comment:
:man_shrugging: Seems to be an issue with `pytest-cov` and we just need to ignore it. It's not actually causing a problem right now